addons/website_jitsi/models/chat_room.py

```diff
@@ -29,6 +29,9 @@ def _default_name(self):
         "Room Name", required=True, copy=False,
         default=lambda self: self._default_name())
     is_full = fields.Boolean("Full", compute="_compute_is_full")
+    jitsi_server_domain = fields.Char(
+        'Jitsi Server Domain', compute='_compute_jitsi_server_domain',
+        help='The Jitsi server domain can be customized through the settings to use a different server than the default "meet.jit.si"')
```

Comment:
> can be customized through the settings

Are we going to add Settings (res.config.settings) for this  :thinking: ?

Reply:
Yes, but in master

Reply:
See https://github.com/odoo/odoo/pull/55869